torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1256,31 +1233,18 @@ Example::
 ProcessGroup options for the NCCL backend
 
 Arguments:
-    timeout (timedelta, optional): Timeout for operations executed against
-            the process group. Default value equals 30 minutes. This is
-            applicable only if the environment variable ``NCCL_BLOCKING_WAIT``
-            or ``NCCL_ASYNC_ERROR_HANDLING`` is set to 1. When
-            ``NCCL_BLOCKING_WAIT`` is set, this is the duration for which the
-            process will block and wait for collectives to complete before
-            throwing an exception. When ``NCCL_ASYNC_ERROR_HANDLING`` is set,
-            this is the duration after which collectives will be aborted
-            asynchronously and the process will crash. `
     is_high_priority_stream (bool, optional): flag to enable/disable process
             group to pick up high priority cuda streams. It lets CUDA driver
             to prioritize NCCL kernels when there are compute kernels waiting.
 
 Example::
     >>> import torch.distributed as dist
-    >>> from datetime import timedelta
     >>>
-    >>> nccl_options = dist.ProcessGroupNCCL.Options(timeout=timedelta(seconds=300))
```

Comment:
Nit: in the doc above (L1236), let's add that is_high_priority_stream is false by default?